luxonis_train/nodes/backbones/ddrnet/ddrnet.py

```diff
@@ -241,8 +241,13 @@ def __init__(
         self.layer5_bottleneck_expansion = layer5_bottleneck_expansion
         self.init_params()
 
-        if download_weights and var.weights_path:
-            self.load_checkpoint(var.weights_path)
+        if download_weights:
+            if var.weights_path:
+                self.load_checkpoint(var.weights_path)
+            else:
+                logger.warning(
+                    f"No checkpoint available for {self.__class__.__name__}, skipping."
```

Comment:
There's the `BaseNode.name` property that can be used instead of `__class__.__name__`.